newrelic/resource_newrelic_nrql_alert_condition_test.go

```diff
@@ -464,19 +468,19 @@ resource "newrelic_nrql_alert_condition" "foo" {
 		duration      = %[4]d
 		operator      = "above"
 		priority      = "critical"
-		threshold     = 1.5
+		threshold     = 1.55
 		time_function = "all"
 	}
 
 	term {
 		duration      = 2
 		operator      = "above"
 		priority      = "warning"
-		threshold     = 1.1
+		threshold     = 1.12
 		time_function = "any"
 	}
 
-	violation_time_limit_seconds = 3600
+	violation_time_limit = "EIGHT_HOURS"
```

Comment:
I think we should probably stick with the deprecated `violation_time_limit_seconds` attribute for this test config since it facilitates testing configurations that contain the deprecated fields.

Reply:
Ah yes, good call.